device-types/Server Technology/PRO3X-C3WG36RL-DQJE2NT2.yaml

```diff
@@ -0,0 +1,187 @@
+---
+manufacturer: Server Technology
+model: PRO3X-C3WG36RL-DQJE2NT2
+slug: server-technology-pro3x-c3wg36rl-dqje2nt2
+part_number: PRO3X-C3WG36RL-DQJE2NT2
+u_height: 0
+is_full_depth: false
+airflow: passive
+comments: PRO3 Switched, Per-Outlet & Per-Inlet Power Sensing, 60A, 208V, (18) C13 & (18) Cx/C19/C13
+console-ports:
+  - name: console/modem
+    label: SER
+    type: rj-45
+    poe: false
+  - name: sensor
+    label: SENSOR
+    type: rj-45
+    poe: false
+  - name: usb-a
+    label: USB-A
+    type: usb-a
+    poe: false
+  - name: usb-B
+    label: USB-B
+    type: usb-b
+    poe: false
```

Comment:
Are these actually used for console access? If not, please remove the definitions

Reply:
Thanks for the feedback, 

USBs can console in (see below) but how do I model the sensor connection? I was going to make sensors devices and connect back to controller located on this device. 

`The PRO3X can emulate a USB-to-serial converter over a USB connection. A USB-to-serial driver named "Dominion PX2 Serial Console" is required for Microsoft® Windows® operating systems.`

Reply:
Great, then the USBs can stay as console ports. As for the sensors, right now we are generally mocking those and fans up as `module-bays`. I'm not at my computer but I can review this more tomorrow morning

Reply:
Awesome thank you, the direction would be great! I have more PRO3x versioned PDUs to add and would use this to template the rest.

Reply:
@DanSheps Hey Dan, care to take a look at this and give your opinion? The sensor ports are RJ45 on the rack PDU, but I am not sure if `console-ports` is the proper area for it.. or if it should be included in the libraries copy at all

Reply:
Hey @danner26, I had been away for a couple of weeks and just re-stepped into this. My thought on this is the sensor port is baked onto the network card in similar fashion to the other connections and would be weird in my opinion to exclude it.

Reply:
[Network card for platform](https://cdn10.servertech.com/assets/documents/documents/885/original/303-9999-51_Replace_NIM8G_Rev_A.pdf?1590772816)

Reply:
It would be an interface IMO, not a console port.

Reply:
Similar, I would say the USB-B is an interface, it is used for diagnostic logging

Reply:
Ok great, would love to merge this so ill kick those items out. Thanks guys